megalinter/reporters/AzureCommentReporter.py

```diff
@@ -107,14 +101,30 @@ def produce_report(self):
             )
             git_client: GitClient = connection.clients.get_git_client()
 
-            # Get repository
-            repository = git_client.get_repository(
-                pr_repo_name
-            )
+            # Get repository id
+            if SYSTEM_PULLREQUEST_SOURCEREPOSITORYURI == "":
```

Comment:
defaut return value for config.get is `None` if a default value is not provided as 3rd parameter

Reply:
Makes sense, corrected in the config.get